ai-platform-unified/notebooks/official/pipelines/google_cloud_pipeline_components_automl_tabular.ipynb

```diff
@@ -0,0 +1,1075 @@
+{
```

Comment:
<p>If I'm not confusing myself-- you changed the package_path name from what I had originally.  If that's the case, how about just naming it <code>tabular_regression_pipeline.json</code>  so that the replace is not necessary?</p><p>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
Update: I see this is construction is in multiple notebooks, e.g. (in a different file)
`package_path="control_pipeline.json".replace(' ', '_')`
I think that might be unnecessary detail for the user, and it would make sense not to add the `replace`? (& just create filename strings that don't need it), if I'm understanding the rationale.

Reply:
I changed the path names to not use shorthand, like classif for classification. The shorthand can be confusing for ESL users.

The names are actually auto-generated.

Reply:
Non-shorthand file names are a good idea, but I think a new user might wonder what's up with the `replace` and why it's necessary.  It seems like extra complexity as a result of 'process' that doesn't help with clarity on the main topic.
How about keeping your process for generating the new names, but then editing the notebooks so the filenames have no spaces and the replace is not necessary?